soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -1614,14 +1614,16 @@ void KaleidoScope_DrawInfoPanel(GlobalContext* globalCtx) {
 
     if ((pauseCtx->cursorSpecialPos == PAUSE_CURSOR_PAGE_LEFT) && (pauseCtx->unk_1E4 == 0)) {
         gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, D_808321A0, D_808321A2, D_808321A4, D_808321A6);
+    } else {
+    	gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, 180, 210, 255, 255);
```

Comment:
Why was this color chosen, instead of `210, 255, 220` from the deleted `gDPSetPrimColor`?

Also, nit: the line should be indented 1 more over.

Reply:
`gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 180, 210, 255, 220);`
is the color I have from decompilation files itself.
the first `gSPDisplayList(POLY_OPA_DISP++, gLButtonIconDL);` do no have that color set before being draw while the second `gSPDisplayList(POLY_OPA_DISP++, gRButtonIconDL);` does have it applied without condition since it's been added right after the gLButtonIconDL draw.
These two else are added to set these gDPSetPrimColor only on hover.

But as I tried to said in main post probably the original intended result was to apply that color on both button without being on hover.
Something is wrong in original stuff and that a chance on two. :)